teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<p>You so nicely explained what to expect in all references, despite</p><blockquote>Pierre-Simon Laplace, Oeuvres Complètes de Laplace. Théorie Analytique des Probabilités (<em>volume VII Gauthier-Villars</em>&nbsp;(1820), 3rd ed)</blockquote><p>Could you add sth here also?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>